wolox-books/src/app/screens/register/register.component.html

```diff
@@ -10,7 +10,7 @@
     </div>
     <div class="section-buttons">
       <button class="button button-register" type="submit" [disabled]="!registerForm.valid">Registro</button>
-      <button class="button button-login" type="button" [disabled]="!registerForm.valid">Login</button>
+      <button class="button button-login" type="button" routerLink="/login">Login</button>
```

Comment:
if you are useing routerLink you should change the router tag to an anchor tag :)